pygdf/series.py

```diff
@@ -7,6 +9,7 @@
 import numpy as np
 
 from . import cudautils, formatting
+from . import dataframe
```

Comment:
This is a circular import as dataframe imports series, but I believe you aren't using this so it can be removed.

Reply:
The import was a remnant of prior code I was trying, forgot about it, will remove it.

Agree with needing the parameter for sort (similar to pandas actually), will add it.

Test for unique_count is already in test_stats (line 59), the same function for unique and value counts:

```assert len(set(arr[mask])) == sr.unique_count()```

Didn't want to add new functions for testing three things that share most of the code, let me know if its alright.